tasks/coverage/src/sourcemap.rs

```diff
@@ -0,0 +1,221 @@
+use crate::suite::{Case, Suite, TestResult};
+use oxc_span::SourceType;
+use oxc_tasks_common::{project_root, TestFiles};
+use std::io::Write;
+use std::{
+    fs::File,
+    path::{Path, PathBuf},
+};
+
+static FIXTURES_PATH: &str =
+    "tasks/coverage/babel/packages/babel-generator/test/fixtures/sourcemaps";
+
+pub struct SourcemapSuite<T: Case> {
+    test_root: PathBuf,
+    test_cases: Vec<T>,
+}
+
+impl<T: Case> SourcemapSuite<T> {
+    pub fn new() -> Self {
+        Self {
+            test_root: project_root().join(FIXTURES_PATH),
+            test_cases: TestFiles::complicated()
```

Comment:
Here the big file the snapshot is very big, maybe we should remove it and add some small test.

Reply:
Agree. Is "react" small enough? You can filter it from tasks/common/src/test_file.rs

Reply:
Updated.